jetty-start/src/main/java/org/eclipse/jetty/start/CommandLineBuilder.java

```diff
@@ -123,17 +170,31 @@ public void addEqualsArg(String name, String value)
     }
 
     /**
-     * Add a simple argument to the command line.
-     * <p>
-     * Will <b>NOT</b> quote/escape arguments that have a space in them.
-     *
-     * @param arg the simple argument to add
+     * @param option the option
+     * @param name the name
+     * @param value the value
      */
-    public void addRawArg(String arg)
+    public void addArg(String option, String name, String value)
     {
-        if (arg != null)
+        if (commandLine.length() > 0)
+            commandLine.append(separator);
+        commandLine.append(option);
```

Comment:
The logic to add `option`, `name` and `value` to `commandLine` and `args` seems different.
Should they not be paired and be equal?
E.g. above we always add `option` to `commandLine`, why not to `args` too?

Reply:
We do.  All three branches below add option to the args one way or another.     The code is a bit convoluted because it is minimal.  Let me make it more explicit even if it duplicates a few lines....

Reply:
is that better?